xml/ReportingOrganisation.xml

```diff
@@ -2922,12 +2922,12 @@
         <codelist-item status="active">
             <code>NL-KVK-41180500</code>
             <name>
-                <narrative>LIGHT FOR THE WORLD The Netherlands</narrative>
+                <narrative>SeeYou Foundation</narrative>
             </name>
             <codeforiati:organisation-type-code>21</codeforiati:organisation-type-code>
             <codeforiati:organisation-type>International NGO</codeforiati:organisation-type>
             <codeforiati:hq-country-or-region>Netherlands</codeforiati:hq-country-or-region>
-            <codeforiati:registry-identifier>lftwnl</codeforiati:registry-identifier>
+            <codeforiati:registry-identifier>syf</codeforiati:registry-identifier>
```

Comment:
I guess this change is okay…

According to a modal currently on the website, the organisation has rebranded:
![Screenshot 2021-06-09 at 08 54 22](https://user-images.githubusercontent.com/464193/121315495-4c2afa00-c900-11eb-972b-233adf873439.png)

Reply:
relatedly, it seems a bit unhelpful that LFTW - **Netherlands** is rebranding into SYF **International** - so not even the same organisation?